views/attendanceStatistics.njk

```diff
@@ -12,23 +12,20 @@
    {%  set toDate = null %}
 {%  endif %}
 
+
 {% block beforeContent %}
-  {{ govukBreadcrumbs({
-  items: [
-    {
-      text: "Home",
-      href: notmUrl
-    },
-    {
-      text: "Manage prisoner whereabouts",
-      href: "/manage-prisoner-whereabouts"
-    },
-    {
-      text: "Attendance reason statistics"
-    }
-  ]
-})
-}}
+    {{ govukBreadcrumbs({
```

Comment:
A little note - there seems to have been a problem with formatting - isn't it always 2 spaces?